### PR TITLE
OSDOCS-10900 adding new required permissions for AWS install

### PR DESCRIPTION
**Pull request by bscott-rh**
Versions:
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10900

Link to docs preview:
https://77511--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-account.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/installation-aws-permissions.adoc [hunk 1]
```diff
@@ -46,6 +46,7 @@ cluster, the IAM user requires the following permissions:
 * `ec2:DescribeNetworkAcls`
 * `ec2:DescribeNetworkInterfaces`
 * `ec2:DescribePrefixLists`
+* `ec2:DescribePublicIpv4Pools`
```
Comment (r4f4) [inline modules/installation-aws-permissions.adoc:49]: Only required if `publicIpv4Pool` is specified in the install-config.

File: modules/installation-aws-permissions.adoc [hunk 2]
```diff
@@ -58,6 +59,7 @@ cluster, the IAM user requires the following permissions:
 * `ec2:DescribeVpcClassicLinkDnsSupport`
 * `ec2:DescribeVpcEndpoints`
 * `ec2:DescribeVpcs`
+* `ec2:DisassociateAddress`
```
Comment (r4f4) [inline modules/installation-aws-permissions.adoc:62]: Only required if `publicIpv4Pool` is specified in the install-config.

Comment (bscott-rh) [reply modules/installation-aws-permissions.adoc]: Updated these 2, thanks.

**Comment by bscott-rh** [comment]
@yunjiang29 PTAL at this doc PR updating AWS permissions for 4.16

**Pull request merged**